.github/workflows/sdk-suggestion.yaml

```diff
@@ -0,0 +1,121 @@
+name: Speakeasy SDK Suggestion Workflow
+
+on:
+  workflow_call:
+    inputs:
+      speakeasy_version:
+        description: The version of the Speakeasy CLI to use or "latest"
+        default: latest
+        required: false
+        type: string
+      openapi_doc_auth_header:
```

Comment:
Exactly what happens for remote specs? Do we put a copy of the file in the repo then make a PR on that? I'm wondering if we might want to just only support local files for now

Reply:
Yes. Basically re-using logic to retrieve openapi document that we're using in generate. Not sure if there's an advantage to splitting out functionality so that we only run on local files for now